client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -124,6 +134,18 @@ class CustomContentTypes extends Component {
 		);
 	}
 
+	renderBlogPostSettings() {
+		const { translate } = this.props;
+		const fieldLabel = translate( 'Blog posts' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **57** times:
`translate( 'Blog Posts' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).